app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,41 @@
+class CommentsController < ApplicationController
+  load_and_authorize_resource except: %i[create]
+
+  def create
+    @resource = Issue.find(params[:issue_id])
```

Comment:
use cancancan

Reply:
used cancan for loading @issue in create action

Reply:
ok, now I am using cancan to load issue for create action

Reply:
where? :)

Reply:
done